src/js/components/AppListItemLabelsComponent.jsx

```diff
@@ -0,0 +1,159 @@
+var classNames = require("classnames");
+var React = require("react/addons");
+
+var OnClickOutsideMixin = require("react-onclickoutside");
+var Util = require("../helpers/Util");
+
+var AppListItemLabelsComponent = React.createClass({
+  displayName: "AppListItemLabelsComponent",
+
+  initialMarginTop: null,
```

Comment:
Member variables are not a good idea with React classes and not suggested.
In this case it works, because you are storing numbers, but objects would be shared by reference between the labels-component-class instances, which would lead to horrible errors.
If you need to store something, you can set the state or send something into a Flux-store, we don't have another mechanism in Marathon UI in the moment.

BTW, I don't think this variable is need here.
Just use a local variable in the according function ```recalculateDropdownPosition```. What's the reason to store the variable out?